cmd/sfncli/sfncli.go

```diff
@@ -127,8 +127,24 @@ func main() {
 
 			// Run the command. Treat unprocessed args (flag.Args()) as additional args to
 			// send to the command on every invocation of the command
-			taskRunner := NewTaskRunner(*cmd, flag.Args(), sfnapi, input, token)
-			if err := taskRunner.Process(taskCtx); err != nil {
+			taskRunner, err := NewTaskRunner(*cmd, flag.Args(), sfnapi, input, token)
+			if err != nil {
+				log.ErrorD("process-start", logger.M{"error": err.Error()})
+
+				_, err = sfnapi.SendTaskFailureWithContext(taskCtx, &sfn.SendTaskFailureInput{
+					Cause:     aws.String(err.Error()),
+					TaskToken: &token,
+				})
+				if err != nil {
+					log.ErrorD("process-failure", logger.M{"error": err.Error()})
+				}
+
+				taskCtxCancel()
+				continue
+			}
```

Comment:
one way to perhaps eliminate this code block: move `input` to the argument list for `Process`

Reply:
That makes sense.  Probably makes sense to move flag.Args() as well.

Reply:
Changed in 99168d736b69ae311df5d5fb820b825147d3c8e0